### ISSUE TITLE
unjitted pretrained models and glint training

### ISSUE THREAD

**Issue by rafale77** [cavalleria/cavaface.pytorch#78]
Thanks for sharing this great work.

I was just trying out your pretrained models for inference and was surprised that you are actually sharing torch jitted models? It is very convenient to reimplement but hard to debug or trying to move to TensorRT, would you mind sharing the unjitted ones?

Also I was wondering if you had plans to train this model:

IR-SE-100 | MS1MV3 | Combined | Softmax

with the Glint dataset to see if it would get even better results than the non SE one?
